### DISCUSSION TITLE
Top Layer with only Perimeters

### DISCUSSION THREAD

**Discussion by KirillNovo** [OrcaSlicer/OrcaSlicer#1444] [General]
In BambuStudio on the top layer are only perimeters, how can I achive that in OrcaSclicer?
![Bambu](https://github.com/SoftFever/OrcaSlicer/assets/55396022/b8452ce1-1043-436f-b96a-d5d381355e59)
![Orca](https://github.com/SoftFever/OrcaSlicer/assets/55396022/06fd9786-7d2b-439c-996f-fe9fbd106e53)

**Comment by KirillNovo** [comment]
For anyone looking: 

![Unbenannt](https://github.com/SoftFever/OrcaSlicer/assets/55396022/7034f7c7-5767-4c45-b54f-833db5d562f2)

**Comment by Festivejelly** [comment]
This only works with classic mode for all top surfaces. Currently only the very top layer on Arachne gets applied.
